### PR TITLE
fix Service.php validation empty error message bug

### PR DESCRIPTION
**Pull request by mishavetl**

### CODE REVIEW

File: src/Ship/Service.php [hunk 1]
```diff
@@ -13,55 +13,58 @@ class Service extends AbstractService
 
     public function send($xml)
     {
-        $response = $this->sendRequest($this->serializer->serialize($xml));
-
+		$response = $this->sendRequest($this->serializer->serialize($xml));
```
Comment (Sevavietl) [inline src/Ship/Service.php]: Padding problem here and so on. Please, fix styles.

**Review by Sevavietl — CHANGES_REQUESTED** [review]
Please, fix code style problems

**Comment by mishavetl** [comment]
It is fixed now. The problem was about mixed up tabs and spaces.

**Comment by Sevavietl** [comment]
@mishavetl, please squash all commits into one.

**Comment by mishavetl** [comment]
All done.

**Comment by Sevavietl** [comment]
@mishavetl, thanks for the PR

**Pull request merged**